src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -971,7 +988,7 @@ public async Task GetReleaseViewModel()
         }
 
         [Fact]
-        public async Task GetReleaseViewModel_NotYetPublished()
+        public async Task GetReleaseViewModel_ReleaseNotYetPublishedHasCorrectPublishedDate()
```

Comment:
Since this test method checks various other things over and above the published date, might be worth just keeping the original test method name.